control-plane/rest/service/src/authentication.rs

```diff
@@ -0,0 +1,122 @@
+use actix_web::{Error, HttpRequest};
+use jsonwebtoken::{crypto, Algorithm, DecodingKey};
+use std::str::FromStr;
+
+use actix_web::http::header::Header;
+use actix_web_httpauth::headers::authorization;
+use once_cell::sync::OnceCell;
+use std::fs::File;
+static JWK: OnceCell<JsonWebKey> = OnceCell::new();
```

Comment:
Don't think we need this, you can add data to the rest server and then retrieve it on the from_request from the BearerToken.

Reply:
Thanks, done - this is now done and has removed the need for `once_cell`.